charts/jaeger/templates/collector-ing.yaml

```diff
@@ -24,21 +19,13 @@ spec:
       http:
         paths:
           - path: {{ $basePath }}
-            {{- if $useNetworkingV1 }}
+            {{- if (include "common.ingress.supportsPathType" $) }}
             pathType: ImplementationSpecific
```

Comment:
I'm unsure about this

Before you only set the `pathType` when you had `v1` and in the same logic used the new ingress.

Now this will be set even on 1.18, as that k8s version already supports the `pathType`

Reply:
@bender316 could we get your thoughts on this (since you implemented the original logic)?

Reply:
Seems like 1.18 already has issues with the current logic according to https://github.com/jaegertracing/helm-charts/pull/279